openwisp_controller/config/migrations/0037_alter_management_ip_help_text.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.1.13 on 2021-08-11 16:12
```

Comment:
@Aryamanz29 we can't do this or we will wreak havoc on existing instances, let's put this migration after all the previous migrations (increase its number to be the last) and avoid changing the other migrations.

Reply:
You are right, We don't need to touch previous migration files :+1: